src/popup/stores/AppStore.ts

```diff
@@ -13,6 +14,8 @@ import SendStore from './SendStore';
 import AddTokenStore from './AddTokenStore';
 
 export default class AppStore {
+  @observable public unexpectedError?: string = undefined;
```

Comment:
can we move this to a different store? some sort of GlobalStore. the `AppStore` should just be a parent store that initializes and holds the references to the sub stores.